typescript/cli/src/commands/core.ts

```diff
@@ -0,0 +1,55 @@
+import { CommandModule } from 'yargs';
+
+import { IsmType } from '@hyperlane-xyz/sdk';
+
+import { createHooksConfigMap } from '../config/hooks.js';
+import { createIsmConfigMap } from '../config/ism.js';
+import { CommandModuleWithContext } from '../context/types.js';
+import { log } from '../logger.js';
+
+import { outputFileCommandOption } from './options.js';
+
+/**
+ * Parent command
+ */
+export const coreCommand: CommandModule = {
+  command: 'core',
+  describe: 'Manage core Hyperlane contracts',
+  builder: (yargs) => yargs.command(config).version(false).demandCommand(),
+  handler: () => log('Command required'),
+};
+
+export const config: CommandModuleWithContext<{
+  ismAdvanced: boolean;
+  ismOut: string;
+  hooksOut: string;
+}> = {
+  command: 'config',
```

Comment:
This breaks from the structure all the other config commands are using. i.e. config -> warp instead of warp -> config.

I know folks were generally more in favor of this structure but we should do one of the other, not both

Reply:
this is why I provided the deprecation notice logs– I will have a final PR that removes all of the old naming conventions (tracked by this issue: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3860)

@yorhodes can you chime in here as well on the overall restructuring

Reply:
yes @jmrossy we are migrating to `hyperlane <module> <action>` structure
I guess I agree we should wholesale migrate, though maybe a nice middleground is "silent" aliasing? ie not showing up in --help but maintaining backwards compatibility with the old paths?

Reply:
Recommend breaking and migrating. Small numbers here:
<img width="555" alt="Screenshot 2024-05-29 at 5 13 10 PM" src="https://github.com/hyperlane-xyz/hyperlane-monorepo/assets/710751/e271c085-bba0-45fd-9bd2-67d046e9630b">

Reply:
Agreed– I've now removed the stale commands in favor of the new ones.